public/index.html

```diff
@@ -6,6 +6,26 @@
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
   <meta name="theme-color" content="#000000">
   <title>Authors Haven</title>
+  <!-- <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet"> -->
+	<link href="https://fonts.googleapis.com/css?family=Righteous%7CMerriweather:300,300i,400,400i,700,700i" rel="stylesheet">
+	<!-- Custom styles for this template -->
+	<!-- <link href="assets/css/mediumish.css" rel="stylesheet"> -->
+	<link rel='stylesheet' href='https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css' type='text/css'>
+	<link rel='stylesheet' href="assets/css/demo 2.css">
+	<!-- <link rel='stylesheet' href="assets/css/form.css"> -->
+	<link href="https://fonts.googleapis.com/css?family=Oswald" rel="stylesheet">
+	<link href="https://fonts.googleapis.com/css?family=Oswald" rel="stylesheet">
+	<link href="https://fonts.googleapis.com/css?family=Playfair+Display" rel="stylesheet">
+	<link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
+	<link href="https://fonts.googleapis.com/css?family=Lato:700" rel="stylesheet">
+	<link rel='stylesheet' id='flotheme_general_css-css' href='assets/css/general.css' type='text/css' media='all' />
```

Comment:
You can import the google font in the specific css file where its required instead if declaring them all here

Reply:
I have added the google fonts to the demo2.sccs file that i use for for the pages